.github/pr_processor.py

```diff
@@ -7,7 +7,16 @@
 from redminelib import Redmine
 
 ORG = "pulp"
-REPOS = ["pulpcore", "pulp_file", "pulp_rpm", "pulp_container"]
+TOOLS = ["pulp_installer", "pulp_rpm_prerequisites"]
+REPOS = [
+    "pulpcore",
+    "pulp_file",
+    "pulp_rpm",
+    "pulp_container",
+    "pulp_ansible",
+    "pulp_deb",
+    "pulp-certguard",
```

Comment:
Did you check with pulp_deb and pulp-certguard teams?

Reply:
No, I didn't

Reply:
@mdellweg @quba42 I'm adding pulp_deb here, it will enable @pulpbot to update redmine to post automatically, and add cherry-pick label if the PR refers to an issue.

If you don't want it, just ping me and I will remove it

Reply:
Personally I was going to wait a bit longer before bringing this up.
That being said, I am fine with adding it now and doing most work with a pulp.plan.io issue.

The final call should be made by @mdellweg.